src/SFA.DAS.RoatpOversight.Web/Extensions/OversightReviewStatusExtensions.cs

```diff
@@ -27,6 +27,31 @@ public static string GetLabel(this OversightReviewStatus status)
             }
         }
 
+        public static string GetSpecificLabel(this OversightReviewStatus status)
+        {
+            switch (status)
+            {
+                case OversightReviewStatus.Successful:
+                    return "Successful";
+                case OversightReviewStatus.SuccessfulAlreadyActive:
+                    return "Successful - already active";
+                case OversightReviewStatus.SuccessfulFitnessForFunding:
+                    return "Successful - fitness for funding";
+                case OversightReviewStatus.InProgress:
+                    return "In progress";
+                case OversightReviewStatus.Rejected:
+                    return "Rejected";
+                case OversightReviewStatus.Removed:
+                    return "Removed";
+                case OversightReviewStatus.Unsuccessful:
+                    return "Unsuccessful";
+                case OversightReviewStatus.Withdrawn:
+                    return "Withdrawn";
+                default:
+                    return "";
```

Comment:
This works. But shouldn't you be using the DisplayAttribute for enums so that the Razor view and other parts of the codebase can bind to the metadata?